.github/workflows/release.yaml

```diff
@@ -11,6 +11,7 @@ on:
 
 jobs:
   release:
+    if: "! contains(github.event.head_commit.message, '[skip ci]')"
```

Comment:
Can we make this job trigger-able with workflow dispatch to test this feature once this initial PR is in?